blog-content/4-best-logging-libraries-for-java.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'The 4 Best Frameworks for Robust Application Insights in Java'
```

Comment:
Can we be more explicit and just call it "logging" instead of "application insights"